sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs

```diff
@@ -95,7 +95,7 @@ public void MultipleInputs()
                 if (document.Status == DocumentTranslationStatus.Succeeded)
                 {
                     Console.WriteLine($"  Translated Document Uri: {document.TranslatedDocumentUri}");
-                    Console.WriteLine($"  Translated to language: {document.TranslatedTo}.");
+                    Console.WriteLine($"  Translated to language: {document.TranslatedToLanguageCode}.");
```

Comment:
```suggestion
                    Console.WriteLine($"  Translated to language code: {document.TranslatedToLanguageCode}.");
```

This should apply to all samples where this phrase is use

Reply:
Done